tfx/components/trainer/component.py

```diff
@@ -119,6 +119,7 @@ def __init__(
       custom_config: Optional[Dict[Text, Any]] = None,
       custom_executor_spec: Optional[executor_spec.ExecutorSpec] = None,
       output: Optional[types.Channel] = None,
+      log_output: Optional[types.Channel] = None,
```

Comment:
as it's the run dir / working dir, the usage may not just for tensorboard logs

let's just call it model run or sth generic